docs/deployment.md

```diff
@@ -118,5 +118,21 @@ To use Knavigator outside the cluster, run
 ```
 In this mode, Knavigator requires the `KUBECONFIG` environment variable or the presence of the `-kubeconfig` or `-kubectx` command-line arguments.
 
-To deploy Knavigator inside the cluster, you would need to create a pod. 
-Details: TBD
+To deploy Knavigator inside the cluster, follow these steps:
+
+1. Configure RBAC:
+   - Create a service account.
+   - Create a cluster role with sufficient permissions to manage the KWOK objects and job resources being tested.
```

Comment:
An easier way would be to bind the cluster-admin role to the service account. I think we can mention it in the docs so users who what an "easier" option can use that. What do you think?

Reply:
done, thanks!